esp32-generic.yaml

```diff
@@ -31,8 +31,9 @@ esp32_ble_tracker:
   scan_parameters:
     interval: 1100ms
     window: 1100ms
-    active: true
-
+    active: true # to minimize RF pollution and save some battery you can set this 
```

Comment:
I don't think that we should add this. If people are interested in this, they can read this in the Bluetooth Proxy documentation. We should guide them to get the best experience.

Reply:
Agreed.